src/Uno.UI/UI/ColorHelper.cs

```diff
@@ -1,26 +1,76 @@
 ﻿using System;
 using System.ComponentModel;
-using Microsoft.UI;
+using System.Linq;
+using DirectUI;
+using WindowsColor = Windows/*Intentional space for WinUI upgrade tool*/.UI.Color;
 
-using Color = global::Windows/*Intentional space for WinUI upgrade tool*/.UI.Color;
+namespace Microsoft.UI;
 
-namespace Microsoft.UI
+public static partial class ColorHelper
 {
-	public static partial class ColorHelper
+	/// <summary>
+	/// Retrieves the display name of the specified color.
+	/// </summary>
+	/// <param name="color">The color to get the name for.</param>
+	/// <returns>The localized display name of the color.</returns>
+	public static string ToDisplayName(WindowsColor color)
 	{
-#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "IS_UNIT_TESTS", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
-		public static string ToDisplayName(Color color)
+		var id = GetColorNameResourceId(color);
+		return DXamlCore.Current.GetLocalizedResourceString($"{id}");
+	}
+
```

Comment:
@ahmed605 Can you please include your methods here as discussed in internal chat?